znoyder/config.yml

```diff
@@ -241,14 +241,14 @@ override:
         vars:
           extra_commands:
             - 'dnf install -y python3-stestr python3-oslotest python3-testscenarios'
-            - 'pip install os-win>=3.0.0 requests-aws>=0.1.4 confluent-kafka kazoo'
+            - 'pip install -c https://opendev.org/openstack/requirements/raw/branch/stable/wallaby/upper-constraints.txt os-win>=3.0.0 requests-aws>=0.1.4 confluent-kafka kazoo'
```

Comment:
Are the upper constraints the same for all openstack projects?

Reply:
Generally they should be the same for a given release of OpenStack (there are few projects that by choice are exceptions and do not follow the upper constraints though, of course, the world cannot be too beautiful place).

Reply:
Didn't know that, thanks!